blackbox/docs/general/ddl/partitioned-tables.rst

```diff
@@ -372,9 +372,19 @@ both existing and future partitions of the table.
     cr> ALTER TABLE parted_table ADD COLUMN new_col string
     ALTER OK, -1 rows affected (... sec)
 
-Unlike regular tables, it is possible to change the number of shards of a
-partitioned table. However, the shard configuration will only be applied on
-**future** partitions.
+
+Changing Number of Shards
+-------------------------
+
+it is possible at any time to change the number of shards
+of a partitioned table. Currently, we support two options.
```

Comment:
I'd change this to sections *Altering whole partitioned table*  and *Altering a single partition*

Reply:
I chose the specific wording in order to stress out the difference (existing vs future).
with your proposal this is not stressed enough (at least not from the title) and the "whole partitioned table" is ab bit misleading as well, right? It only applies for the future partitions.

My reasoning was to make that crystal clear from the start.


Let me know your thoughts